resources/eventing/values.yaml

```
@@ -8,7 +8,7 @@ global:
       version: PR-11920
     eventing_controller:
       name: eventing-controller
-      version: 571bf38e
+      version: PR-12553
```

```suggestion
      version: PR-12599
```
The idea of an image bump is to use the exact image that was created by this PR